src/lib/sesame.service.spec.ts

```diff
@@ -45,12 +47,20 @@ describe('SesameService', () => {
   jwtUtils.validate = () => {
   };
 
+  const mockAuthService = {
+    authorizeThenRedirect() {},
+    hasAnyRoles(): boolean {
+      return true;
+    },
+  };
+
```

Comment:
Dans le cas où d'autres tests utilisant mockAuthService seraient ajoutés à l'avenir.

```suggestion
 const mockAuthService = {
    authorizeThenRedirect: jasmine.createSpy("authorizeThenRedirect"),
    hasAnyRoles: jasmine.createSpy("hasAnyRoles")
  };

  mockAuthService.authorizeThenRedirect.and.returnValue({});
  mockAuthService.hasAnyRoles.and.returnValue(true);
```

Reply:
ok !